tests/unit/TranslateTest.js

```diff
@@ -52,3 +54,38 @@ describe('translate', () => {
         expect(translate.translate('en', ['testKeyGroup', 'testFunction'], {testVariable})).toBe(expectedValue);
     });
 });
+
+describe('Translation Keys', () => {
+    let activeLanguage;
+    let path = '';
+    function matchKeys(source, target, key) {
+        path += key ? `${key}.` : '';
+        const pathLevel = path;
+        if (key && !_.has(target, key)) {
+            console.debug(`🏹 ${path.slice(0, -1)} is missing from ${activeLanguage}.js`);
```

Comment:
Instead of a debug log line, can we make it so that the test itself fails? Ideally it would show one failed test per missing key if that's possible, but if that's hard we can have one test fail that prints all missing keys.

Reply:
Test will fail, if key don't match.  This log is just to show what is missing. 

> Ideally it would show one failed test per missing key if that's possible,

I don't think its a good idea to create multiple tests that basically test the same thing. 

> if that's hard we can have one test fail that prints all missing keys.

This can be done. We can log all the missing keys.

Reply:
ok cool, my main point is that it's better to have the test failure show what keys failed instead of having to look at the logs.

Reply:
I tried to find way to add the missing key as failure message itself but I don't think its possible AFAIK. So we have to stick with logs.

Reply:
hmmmm, I think that if you change the expect from `toBeTruthy` to something like `arrayContaining` would make jest to print the diff of the arrays and thus the missing keys?